### DISCUSSION TITLE
Request: child tab order option

### DISCUSSION THREAD

**Discussion by SteveOnorato** [Able-Bear-Software/tab-nodes-tree#17] [Ideas / Feature Request]
On page A, I center-click on links to open page B, then page C, then page D.

Without the extension, Chrome orders the tabs A, B, C, D.  This is logical.
But with Tab Nodes Tree, the order is A, D, C, B.  This messes with me.
Can you have an option to preserve the default behavior (keep child tabs in the order opened)?
Ideally the tree would be:
A
|- B
|- C
|- D

Thanks!

**Comment by zeroarst** [comment]
Hey @SteveOnorato,

You're absolutely right—I changed the default behavior to move newly opened tabs to the first child position in the tree. I understand how this can be disorienting. I'll add an option that allows users to choose between the Chrome default order (keeping child tabs in the order they were opened) or placing new tabs as the first child of the subtree.

Thanks for the feedback!

**Reply by zeroarst** [reply]
Ticket created: https://github.com/Able-Bear-Software/tab-nodes-tree/issues/20

**Comment by zeroarst** [comment]
This issue was expected to be resolved in version 1.3.0. Please let me know if you encounter any problems, and I can reopen it for further assistance.
